interpreter/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ else()
 endif()
 set(CLING_PLUGIN_INSTALL_PREFIX "${etcdir}/cling/plugins/" CACHE STRING "" FORCE)
 set(LLVM_DIR "${CMAKE_BINARY_DIR}/interpreter/llvm/src")
-set(CLING_BUILD_PLUGINS OFF)
+set(CLING_BUILD_PLUGINS ON)
```

Comment:
@amadio ping.

Reply:
I don't understand the ping, but in any case, clad should be under an `option(clad)` and not forcibly enabled by default, because it does a git clone, and will fail if one downloads ROOT sources in a tarball and don't have git installed, or if ROOT is built inside a network sandbox (which is common for package managers to do). Nothing that downloads stuff should be hard-coded to `ON`. This is a pre-requisite for me to be able to solve [ROOT-9163](https://sft.its.cern.ch/jira/browse/ROOT-9163).